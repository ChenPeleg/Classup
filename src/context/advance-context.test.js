import React from "react";
import { shallow, mount, render } from "enzyme";
import AdvanceContext from "./advance-context";
const dummyContext = (<><AdvanceContext.Consumer>
  {(context) => (
    <p>{context.questionNumber}</p>)} </AdvanceContext.Consumer></>)
describe("<advance-context >", () => {
  it("renders correctly", () => { expect(dummyContext).toMatchSnapshot() });

})