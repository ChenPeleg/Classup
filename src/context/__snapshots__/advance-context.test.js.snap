// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<advance-context > renders correctly 1`] = `
<React.Fragment>
  <Context.Consumer>
    [Function]
     
  </Context.Consumer>
</React.Fragment>
`;
